articles/api/management/v2/create-m2m-app.md

```diff
@@ -25,9 +25,15 @@ To create and authorize a Machine-to-Machine Application for the Management API:
 The application created in the steps above has been granted __all__ the Management API <dfn data-key="scope">scopes</dfn>. This means that it can access all endpoints.
 
 ::: panel How can I find out which scopes/permissions are required?
-Each machine-to-machine application that accesses an API must be granted a set of <dfn data-key="scope">scopes</dfn>. Scopes are permissions that should be granted by the owner. Each [Auth0 Management API v2](/api/management/v2) endpoint requires specific scopes. To see the required scopes/permissions for each endpoint, go to the [Management API Explorer](/api/management/v2#!) and find the endpoint you want to call. Each endpoint has a section called **Scopes** listing all the scopes that the endpoint requires. For example, the [Get all clients](/api/management/v2#!/Clients/get_clients) endpoint requires the scopes `read:clients` and `read:client_keys`.
+Each [Auth0 Management API v2](/api/management/v2) endpoint requires specific scopes in the machine-to-machine token in order to get the data you want. Each endpoint in the [Management API Explorer](/api/management/v2#!) documentation has a section called scopes, listing all the scopes that the endpoint requires.
 :::
 
+## Example: Get All Connections Endpoint
+
+The [Get all connections](/api/management/v2#!/Connections/get_connections) endpoint requires the `read:connections` scope, whilst the [Create a connection](/api/management/v2#!/Connections/post_connections) endpoint requires the `write:connections` scope. Using that we now know that our machine-to-machine token should only require the `read:connections` scope in order to get that data.
+
+We do recommend that if you have multiple applications that need access to the Management API that you create separate machine-to-machine applications for each application in Auth0 instead of one.
```

Comment:
Here I also find the paragraph starting with `If you have multiple applications that should access the Management API...` preferable, perhaps because it includes an example.